egs/babel/s5d/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,336 @@
+#! /bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# Features configs
+feat_type=plp
+feat_config=conf/plp.conf
+add_pitch=true
+pitch_config=conf/pitch.conf
+
+# Uniform segmentation options
+max_segment_duration=10
+overlap_duration=2.5
+max_remaining_duration=5
```

Comment:
A comment explaining max_remaining_duration would be nice, as it' snot immediately clear what it is.